### PR TITLE
doc(neo4j): Document supported types

### PR DESCRIPTION
**Pull request by disq**
A bit like [Gremlin](https://github.com/cloudquery/cloudquery/pull/10913) but more pure.

### CODE REVIEW

**Review by hermanschaaf — APPROVED** [review]
Impressive type coverage 👍

**Comment by disq** [comment]
> Impressive type coverage 👍

@hermanschaaf it's just `.ValueStr(i)` tbh

File: website/pages/docs/plugins/destinations/neo4j/types.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+| Interval[Month]        | ✅ Yes      | `String`   |
+| JSON                   | ✅ Yes      | `String`   |
+| Large Binary           | ✅ Yes      | `Bytes`    |
+| Large List             | ✅ Yes      | `List`     |
+| Large String           | ✅ Yes      | `String`   |
+| List                   | ✅ Yes      | `List`     |
+| MAC                    | ✅ Yes      | `String`   |
+| Map                    | ✅ Yes      | `String`   |
+| String                 | ✅ Yes      | `String`   |
+| Struct                 | ✅ Yes      | `String`   |
```
Comment (hermanschaaf) [inline website/pages/docs/plugins/destinations/neo4j/types.md:38]: Should we document how the struct is actually encoded into string?

Comment (disq) [reply website/pages/docs/plugins/destinations/neo4j/types.md:38]: Yes maybe https://github.com/cloudquery/cloudquery/pull/10955

**Pull request merged**